cluster-install/install.sh

```diff
@@ -2156,15 +2187,15 @@ step_generate_trident_fresh_install_yaml() {
     local enable_acp="true"
     local labels_field_and_content=""
     if [ -n "$IMAGE_PULL_SECRET" ]; then pull_secret='["'$IMAGE_PULL_SECRET'"]'; fi
-    if [ -n "$PROCESSED_LABELS" ]; then
+    if [ -n "$_PROCESSED_LABELS" ]; then
```

Comment:
This was missing the underscore `_`, which meant the variable was always empty, which in turn prevented labels from being applied to the TORC.